neural_compressor/adaptor/torch_utils/smooth_quant.py

```diff
@@ -16,28 +16,25 @@
 # limitations under the License.
 #
 from neural_compressor.utils.utility import LazyImport
+import json
 
 torch = LazyImport('torch')
 from ...utils import logger
 
 
 def model_forward(model, dataloader, iters):
+    from ..pytorch import pytorch_forward_wrapper
     try:
         cnt = 0
         for idx, (input, label) in enumerate(dataloader):
-            output = model(input)
+            output = pytorch_forward_wrapper(model, input)
```

Comment:
is any issue here? I'm not too fond of coupling this with INC's other module here.  Add  another "device" parameter should be enough

Reply:
It's more stable than the older one with more condition judgement. Reusing table function is recommended, I think